src/lib/y2network/sysconfig/interface_file.rb

```diff
@@ -278,7 +284,14 @@ def save
       #
       # @return [String] Interface's type depending on the file values
```

Comment:
Are we returning a string?

Reply:
According to my last changes should be an `InterfaceType`, so.. updating it.